src/ceph-disk/ceph_disk/main.py

```diff
@@ -2257,26 +2262,34 @@ def create_key(self):
         key_size = CryptHelpers.get_dmcrypt_keysize(self.args)
         key = open('/dev/urandom', 'rb').read(key_size / 8)
         base64_key = base64.b64encode(key)
-        command_check_call(
-            [
-                'ceph',
-                'config-key',
-                'put',
-                'dm-crypt/osd/' + self.args.osd_uuid + '/luks',
-                base64_key,
-            ],
-        )
-        keyring, stderr, ret = command(
-            [
-                'ceph',
-                'auth',
-                'get-or-create',
-                'client.osd-lockbox.' + self.args.osd_uuid,
-                'mon',
-                ('allow command "config-key get" with key="dm-crypt/osd/' +
-                 self.args.osd_uuid + '/luks"'),
-            ],
-        )
+        config_key_cmd = [
```

Comment:
nit, could also have a variable named "ceph_cmd", set it to `['ceph']` `if self.args.id`, and `['ceph', '--id', self.args.id]`, otherwise.
and a variable named `config_key_args` with the args for "config-keys". and then call `command_check_call(ceph_cmd + config_key_args)`.